### DISCUSSION TITLE
which type of file extension will osu!lazer's installer use?

### DISCUSSION THREAD

**Discussion by Modsnotworking** [ppy/osu#16324] [Q&A] [answered]
Is it going to be .exe or .msi
I don't know if msi[file extension] is only appplicable to microsoft products. but I think msi[file extension] is also available for third party software

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu#running-osu

**Reply by bdach** [reply]
i cannot for the life of me figure out why you would need that information??? rest assured that it will work. regardless of the final file extension.
